### ISSUE TITLE
Scatter stuck infinite loading when failing to import invalid seed words

### ISSUE THREAD

**Issue by Soleone** [GetScatter/ScatterDesktop#505]
## Problem

Some users try to import old Scatter words (backup phrase) into new Scatter 12.1 in hope to restore private keys (without actually having a backup).

Scatter just gets stuck in an infinite loading screen and doesnt do anything.

![image](https://user-images.githubusercontent.com/2923/82136667-09225400-97de-11ea-878d-33e8d824fd3d.png)

## Suggestion

Ensure the process completes. Try to maybe display an error message to the user to clarify how old scatter words are incompatible and require a backup file.

**Comment by aaronmboyd** [comment]
I'm experiencing this issue after using Scatter 2 years ago and coming back to interact with my EOS for the first time. 

What is the workaround? I had always thought seed words were the complete master backup?

**Comment by ElderOrb** [comment]
> I'm experiencing this issue after using Scatter 2 years ago and coming back to interact with my EOS for the first time.
> 
> What is the workaround? I had always thought seed words were the complete master backup?

+1 here. I can't restore scatter account by seed because of the infinite loading and can't find owner public key :(. As for the scatter backups - it doesn't seem to contain all the info required to recover account. So seed phrase seems to be the only way to recover wallet access.

**Comment by aaronmboyd** [comment]
@ElderOrb I got around it by going back through the releases page and downloading a binary from the past. The version that worked for me was `linux-scatter-10.1.0-x86_64`

**Comment by ElderOrb** [comment]
@aaronmboyd that's what I tried too - but wasn't so lucky... :( based on the timestamp - seed backup was done in June, 2019... I've tried a few versions before that date - doesn't work.

**Comment by aaronmboyd** [comment]
@ElderOrb Yeah sorry no other ideas really. I washed my hands of this protocol a long time ago. Maybe one of the Scatter devs can help out here.
